Appraisals

```diff
@@ -0,0 +1,11 @@
+appraise "rails4.0" do
```

Comment:
Hmm, according to #13, we have a Rails 4 dependency. We could easily change that though if we'd like. I think this also means that we'd have to change the name of the `config.serve_static_files` flag in the dummy app to the Rails 3-compatible one (`server_static_assets` - deprecated in 4). I'm good either way.